src/Broadway/Bundle/BroadwayBundle/Command/SchemaEventStoreCreateCommand.php

```diff
@@ -55,9 +55,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $eventStore    = $this->getEventStore();
 
             $table = $eventStore->configureSchema($schema);
-            $schemaManager->createTable($table);
-
-            $output->writeln('<info>Created schema</info>');
+            if (null !== $table) {
+                $schemaManager->createTable($table);
+                $output->writeln('<info>Created schema</info>');
```

Comment:
As we don't know the context this may be run in, the message might be a bit unclear.
What about `Created Broadway event-store schema`?